moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -0,0 +1,303 @@
+from pathlib import Path
+from typing import Optional, List
+import logging
+from ament_index_python.packages import get_package_share_directory
+
+from parameter_builder import ParameterBuilder, load_yaml, load_xacro
+
```

Comment:
Can we add inline documentation at the top describing how this file is used, please? Just copy Tyler's first comment in this PR with his example code.